pootle/core/utils/db.py

```diff
@@ -34,11 +34,11 @@ def set_mysql_collation_for_column(apps, cursor, model, column, collation, schem
     if current_collation != collation:
         # set collation
         cursor.execute(
-            "ALTER TABLE %s.%s"
```

Comment:
The migrations passed with the changes in this PR. But I agree it won't harm using backticks there too. Updating PR.